src/components/BlueHint/index.js

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+
+import style from './style.scss';
+
+const BlueHint = ({
+  firstLine, secondLine, className,
```

Comment:
Instead of passing each line separately it could be an array of texts.

Reply:
👍